src/App.jsx

```diff
@@ -0,0 +1,78 @@
+import {useState} from 'react'
+import "./App.css";
+import Contacts from './contacts.json'
+
+
+
+function App() {
+
+  const contactList = Contacts.slice(0,5)
+  const remainingContacts = Contacts.slice(5)
+  // console.log(remainingContacts)
+  // console.log(contactList)
+
+  const [contacts, setContacts] = useState(contactList)
```

Comment:
Here in this case in order to no get repeated contacts we can do something like this 

```suggestion
  const contactList = Contacts.slice(0, 5)

  const [contacts, setContacts] = useState(contactList)
  const remainingContacts = Contacts.filter((mainDataContacts) => !contacts.some((currentContactFromList) => mainDataContacts.id == currentContactFromList.id))
```
What we are doing here is filtering from the Contacts array that is the JSON file, the items that doenst have the same id form our contacts state